src/web/mjs/connectors/DoujinDesu.mjs

```diff
@@ -7,13 +7,38 @@ export default class DoujinDesu extends WordPressMangastream {
         super.id = 'doujindesu';
         super.label = 'DoujinDesu';
         this.tags = ['hentai', 'indonesian'];
-        this.url = 'https://doujindesu.id';
-        this.path = '/komik-list/?list';
+        this.url = 'https://doujindesu.xxx';
+        this.path = '/komik-list/page/%PAGE%/';
 
         this.querMangaTitleFromURI = 'div#infoarea div.post-body h1.entry-title';
-        this.queryMangas = 'div.listttl ul li a';
+        this.queryMangas = '#main .relat .animepost .animposx a';
         this.queryChapters = 'div#chapter_list div.epsleft span.lchx a';
         this.queryChaptersTitle = undefined;
         this.queryPages = 'div.reader-area img[src]:not([src=""])';
     }
+
+    // NOTE: Fallback to pagination since '/komik-list/?list' empty
+    async _getMangas() {
+        let mangaList = [];
+        let path;
+        for(let page = 1, run = true; run; page++) {
+            path = this.path;
+            if(page == 1) path = '/komik-list/';
+            let mangas = await this._getMangasFromPage(path, page);
```

Comment:
Consider using ternary operator
`this._getMangasFromPage(page > 1 ? this.path : '/komik-list/', page)`